tests/modules/og_test/og_test.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Organic Groups test'
+type: module
+description: 'Support module for Organic Groups testing.'
+package: Testing
+version: VERSION
```

Comment:
This `version` property shouldn't be there. That's just a placeholder. When a release is made this it added automatically by the packager on drupal.org.